src/lib/clients/IOClients/apps/Builder.ts

```diff
@@ -47,9 +48,11 @@ export class Builder extends AppClient {
   }
 
   private stickyHost!: string
+  private linkID: string
 
   constructor(ioContext: IOContext, opts?: InstanceOptions) {
     super('vtex.builder-hub@0.x', ioContext, opts)
+    this.linkID = randomBytes(20).toString('hex')
```

Comment:
I think 20 bytes for this is maybe too much - 20 bytes = 160 bits = 10^48. I think 32, or 64 bits, is enough, so 4bytes or 8 bytes